### PR TITLE
Ashwalker shaman stuff

### PR DESCRIPTION
**Pull request by Marmio64**
Adds a shaman headdress with the same stats and crafting recipe as the skull helmet, just looks different. Tribal headdress however doesn't cover the face or anything, so you can eat and stuff through it. Sprite is from the ss13 fallout servers. 

Also adds Tribalordrazine (I am not good with names), which is a slightly weaker tricord that can be made by mashing together mushroom stems, cactus fruit, ash (from ashpiles you can get from burning stuff in a bonfire), and then heating it with a welder. Might add more stuff if people want, but I also don't want ashwalkers to be too snowflakey or anything.

edit: Adds Capmix too. Capmix is like charcoal, but instead of getting rid of reagents normally, it forces you to vomit it up. You can make it from mashing the poisonous mushroom caps and ash together, then heating it with a welder. The reaction will remove the toxins, but you'll still get hallucinations from the mindbreaker and mushroom hallucinogens.

shamanlizard
picture of an ashwalker with the shaman hat on

#### Changelog

:cl:  
rscadd: Adds a mixable tribal variant of tricord, and a shaman headdress.
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
more ashwalker content

**Comment by cuackles** [comment]
You could name the chem something like healing salve or necropolis blessing, something that's sounds primitive i suppose

**Comment by Marmio64** [comment]
> You could name the chem something like healing salve or necropolis blessing, something that's sounds primitive i suppose

salves are something you use on the skin though and necropolis blessing sounds like something that is more magical, meanwhile the chem is more just practical science or whatever

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
@Partheo Merge at your discretion

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -299,3 +299,19 @@
 	mix_message = "The solution partially dissolves, binding the remaining ingredients."
 	results = list(/datum/reagent/medicine/burnmix = 3) // fuel and most ethenol burns away to form BurnMix from remaining ethenol and ash
 	required_reagents = list(/datum/reagent/fuel = 1, /datum/reagent/consumable/ethanol/beer = 3, /datum/reagent/ash = 2) // Take a booze bottle, drink some , add welding fuel and ash. Heat up with lighter
+
+/datum/chemical_reaction/tribalordrazine // Mushroom stem + ash from an ash pile + cactus fruit + heat the bowl with a welder
+	name = "Tribalordrazine"
+	id = /datum/reagent/medicine/tribalordrazine
+	required_temp = 420
+	mix_message = "The mixture binds together from the heat, releasing a pleasent smelling smoke."
```
Comment (SomeguyManperson) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: Pleasant

**Review by Partheo — APPROVED** [review]

**Pull request merged**